client/landing/stepper/declarative-flow/migration/index.tsx

```diff
@@ -181,7 +181,9 @@ const useCreateStepHandlers = ( navigate: Navigate< StepperStep[] >, flowObject:
 						platform: 'wordpress',
 						siteId,
 						siteSlug,
-						backToStep: MIGRATION_UPGRADE_PLAN,
+						backToStep: PLATFORM_IDENTIFICATION,
+						migrateEntireSiteStep: MIGRATION_UPGRADE_PLAN,
+						replaceHistory: true,
```

Comment:
Applying [this suggestion](https://github.com/Automattic/wp-calypso/pull/93736#discussion_r1723820480) here.